Patches.py

```diff
@@ -2262,8 +2395,13 @@ def get_override_entry(location):
     elif location.type == 'Chest':
         type = 1
         default &= 0x1F
-    elif location.type == 'Collectable':
+    elif location.type == 'ActorOverride':
         type = 2
+    elif location.type == 'Collectable':
+        if any(kind in location.filter_tags for kind in ("Pot", "Crate", "Drop", "FlyingPot", "SmallCrate", "RupeeTower", "Beehive")):
```

Comment:
Why use filter tags and stuff instead of just making the location type something more specific than 'Collectable'?

Reply:
That's just how I originally wrote it because it reuses some of the hacks from the Collectable hack used for heart pieces and keys. Then it kinda snowballed out of control. I always intended on going back to change it. I guess I can do that before the merge.